framework/Source/BasicOperation.swift

```diff
@@ -76,10 +76,17 @@ open class BasicOperation: ImageProcessingOperation {
                 uniformSettings["aspectRatio"] = firstInputTexture.aspectRatio(for: outputRotation)
             }
             
-            guard let commandBuffer = sharedMetalRenderingDevice.commandQueue.makeCommandBuffer() else {return}
+            guard 
+                let commandBuffer = sharedMetalRenderingDevice.commandQueue.makeCommandBuffer(),
+                let outputTexture = Texture(
+                    device:sharedMetalRenderingDevice.device,
+                    orientation: .portrait,
+                    width: outputWidth,
+                    height: outputHeight,
+                    timingStyle: firstInputTexture.timingStyle
+                )
+            else { return }
```

Comment:
suggestion: We could cosndier add an assertion for it

Reply:
No problem; I'll add it later on, and so are other similar cases.